client/src/integration/java/com/networknt/client/Http2ClientIT.java

```diff
@@ -674,6 +674,29 @@ public String callApiAsync() throws Exception {
         return reference.get().getAttachment(Http2Client.RESPONSE_BODY);
     }
 
+    public ByteBuffer callApiWithByteBuffer() throws Exception {
```

Comment:
Hi! After manual testing, the byte array response can be handled properly! But this test method seems not be invoked and no @test annotation. Does this need to be added?

Reply:
I will take a look